tests/Compat/CompatFormTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+use ipl\Tests\Web\TestCase;
+use ipl\Web\Compat\CompatForm;
+
+class CompatFormTest extends TestCase
+{
+    /** @var CompatForm */
+    private $form;
+
+    protected function setUp(): void
+    {
+        $this->form = new CompatForm();
+    }
+
+    public function testSubmitButtonPrefixApplied(): void
+    {
+        $submitButton = $this->form->createElement('submit', 'submitCreate');
```

Comment:
I know why you're not using `addElement` here. But I think the order of submits is not relevant for this test. Please do use it, to reduce complexity.